pkg/trace/stats/quantile/ddsketch_converter.go

```diff
@@ -0,0 +1,180 @@
+package quantile
+
+import (
+	"errors"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/sketches-go/ddsketch/mapping"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/gogo/protobuf/proto"
+	"sort"
+)
+
+type ddSketch struct {
```

Comment:
`gofmt` and some minimal docs would be welcome. Not sure if needed here or if the reader who is aware of DDSketch should automatically understand this. If it's the latter no worries.